aiida/cmdline/params/options/commands/code.py

```diff
@@ -40,7 +41,9 @@ def is_not_on_computer(ctx):
     prompt_fn=is_on_computer,
     type=types.AbsolutePathParamType(dir_okay=False),
     cls=InteractiveOption,
-    help=('[if --on-computer]: the absolute path to the executable on the remote machine.')
+    help=(
+        '[if --on-computer]: Absolute path to the executable on the target computer (including the executable itself).'
```

Comment:
Again, this was merged in from the manual docs.
I do agree that the parenthesis is not strictly needed - if you prefer to remove it, I would stick with `Absolute path` - I would not know what a `Full` means in this context.

Reply:
I agree that absolute path should be sufficient. I just figured that since the parenthesis were added, it was apparently presumed not to be enough for all users. I'd be happy to remove "Full"